src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectivityTests/AADConnectionTest.cs

```diff
@@ -453,6 +453,27 @@ public static void ActiveDirectoryManagedIdentityWithCredentialsMustFail()
             Assert.Contains(expectedMessage, e.Message);
         }
 
+        [ConditionalFact(nameof(IsAADConnStringsSetup))]
+        public static void ActiveDirectoryWorkloadIdentityWithCredentialsMustFail()
```

Comment:
I still need to write test cases for auth when the test environment supports workload identity, but I need to do more research in how to write that.

Reply:
Sorry, kind of new here, but do you have an update on this? It's 2 months old.

Reply:
Our automated test environment is unlikely to ever support workload identity unless there is a way to emulate it without setting up an expensive Kubernetes environment.

@wsugarman - Do you think that (emulation) could ever happen?

Either way, I'm okay with manual testing of this feature for now.

Reply:
I'm unaware of any existing emulation of workload identity, but perhaps it may make its way into other AKS-based offerings that could be simpler to set-up, like Azure Container Apps? I think in the meantime, without AKS, it will need to be manually tested